src/js/angular/clustermanagement/cluster-drawing.service.js

```diff
@@ -40,11 +41,13 @@ export function setCreateClusterZone(hasCluster, clusterZone, translationsMap) {
             .attr('y', -50)
             .classed('h2', true)
             .style('text-anchor', "middle");
-        textGroup.append('text')
-            .attr('id', 'create-cluster-label')
-            .classed('h3', true)
-            .text(translationsMap.create_cluster_btn || 'Click here to create a cluster')
-            .style('text-anchor', "middle");
+        if (hasAccess ){
+            textGroup.append('text')
+                .attr('id', 'create-cluster-label')
+                .classed('h3', true)
+                .text(translationsMap.create_cluster_btn || 'Click here to create a cluster')
```

Comment:
shouldn't this be translated?

Reply:
I guess they are some sort of fallbacks that got to the master.

I removed them.

All translations are in the `translationsMap`